README.md

```diff
@@ -42,6 +42,9 @@ To limit the build to a single language subtree, *i.e.* English, use
 ```
 make dirhtml LANGUAGES=en
 ```
+
+#### After adding a new entry to package index , Run the github action fortran_packages before building the sphinx build.
```

Comment:
This is a regular sentence, isn't it?

```suggestion
After adding a new entry to package index, run the github action *fortran\_packages* before building the sphinx build.
```

Reply:
This is a regular senetence but to draw the significant attention and to avoid missing them I had done this , But this seems good.